website/docusaurus.config.js

```diff
@@ -60,6 +60,17 @@ module.exports = {
         releaseDate: "2021-03-12",
         firstDocPath: "/apisix",
       },
+      {
+        name: "APISIX™ Docker",
+        nameInParamCase: "docker",
+        description: "The docker for Apache APISIX.",
```

Comment:
What about "Docker tooling for Apache APISIX."

Reply:
Sure :)